Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -585,10 +589,50 @@ namespace AZ
+
+                // Get the render pipeline associated with the shadow pass of the given view
+                RenderPipelinePtr renderPipeline = scene.GetRenderPipeline(view.GetShadowPassRenderPipelineId());             
+                //Only apply this optimization if you only have one view available.
+                if (renderPipeline && renderPipeline->GetViews(renderPipeline->GetMainViewTag()).size() == 1)
+                {
+                    RPI::ViewPtr cameraView = renderPipeline->GetDefaultView();
+                    const Matrix4x4& cameraWorldToClip = cameraView->GetWorldToClipMatrix();
+                    worklistData->m_cameraFrustum = Frustum::CreateFromMatrixColumnMajor(cameraWorldToClip);
+                    worklistData->m_applyCameraFrustumIntersectionTest = true;
+                }
             }
             
             auto nodeVisitorLambda = [worklistData, taskGraph, parentJob, &worklist](const AzFramework::IVisibilityScene::NodeData& nodeData) -> void
             {
+                // For shadow cascades that are greater than index 0 we can do another check to see if we can reject any Octree node that do not
+                // intersect with the camera frustum. We do this by checking for an overlap between the camera frustum and the Obb created
+                // from the node's AABB but rotated and extended towards light direction. 
+                if (worklistData->m_applyCameraFrustumIntersectionTest && worklistData->m_hasExcludeFrustum)
+                {
+                    // Build an Obb from the Octree node's aabb
+                    AZ::Obb extrudedBounds = AZ::Obb::CreateFromAabb(nodeData.m_bounds);
+
+                    // Rotate the Obb in the direction of the light
+                    AZ::Quaternion directionalLightRot = worklistData->m_view->GetCameraTransform().GetRotation();
+                    extrudedBounds.SetRotation(directionalLightRot);
+                    
+                    AZ::Vector3 halfLegth = 0.5f * nodeData.m_bounds.GetExtents();
+                    // After converting AABB to OBB we apply a rotation and this can incorrectly fail intersection test. If you have an OBB cube built from an octree node,
+                    // rotating it can cause it not encapsulate meshes it encapsulated beforehand. The type of shape we want here is essentially a capsule that starts from the
+                    // light and wraps the aabb of the octree node cube and extends towards light direction. This capsule's diameter needs to the size of the body diagonal
+                    // of the cube. Since using capsule shape will make intersection test expensive we simply expand the Obb to have each side be atleast the size of the body diagonal
+                    // which is sqrt(3) * side size. Hence we expand the Obb by 73%. 
+                    halfLegth += halfLegth * Vector3(0.36f, 0.36f, 0.36f);
```

Comment:
```suggestion
                    // which is sqrt(3) * side size. Hence we expand the Obb by 73%. Since this is half length, we expand it by 73% / 2, or 36.5%.
                    halfLength *= Vector3(1.365f);
```